plugins/xtlua_fms/scripts/B777.20.xt.fms/B777.fms.pages.lua

```diff
@@ -1250,7 +1269,7 @@ function fmsFunctions.setdata(fmsO,value)
 		if validate_sethdg(fmsO["scratchpad"]) == false then
 			fmsO["notify"]="INVALID ENTRY"
 		else
-			if fmsModules["data"] ~= "---`" then
+			if fmsModules["data"] ~= "---`" then -- what?
```

Comment:
always nil
heading entry isn't even necessary in the 777, need to clean up unnecessary 747 code.